cis/plugins/validation/mozilliansorg_publisher_plugin.py

```diff
@@ -36,8 +36,12 @@ def run(publisher, user, profile_json):
         'groups'
     ]
 
-    # Check for any non-whitelisted attribute modification
-    # Note that no extra attribute may be added at this stage as we already performed schema validation
+    # Check the easiest case.  None type.
```

Comment:
@andrewkrug  I will be unavailable for most of next week - my vote would be to let mozilliansorg create users when they dont exist only, and until auth0/workday publishers exist. Happy to talk tomorrow though.